en/getting_started/rc_transmitter_receiver.md

```diff
@@ -1,81 +1,109 @@
 
 ![RC Basic Commands](../../images/rc_basic_commands.png)
 
-There are numerous possible layouts for the transmitter's control sticks, switches, etc. The more common layouts have been given specific "Mode" numbers. *Mode 1* and *Mode 2* transmitters (shown below) differ only in the placement of the throttle. 
+There are numerous possible layouts for the control sticks, switches, etc. The more common layouts have been given specific "Mode" numbers. *Mode 1* and *Mode 2* (shown below) differ only in the placement of the throttle. 
 
 ![Mode1-Mode2](../../images/mode1_mode2.png)
 
-> **Note** The choice of transmitter mode is largely one of taste, and must be made before you buy the transmitter.
+> **Note** The choice of mode is largely one of taste (*Mode 2* is more popular).
   
 
-## Transmitters for Ground Vehicles
+## Remote Control Units for Ground Vehicles
 
 An Unmanned Ground Vehicle (UGV)/car minimally requires a 2 channel transmitter in order to send the values for steering and speed. Commonly transmitters set these values using a wheel and trigger, two single-axis control sticks, or a single dual-axis control stick.
 
 There is nothing to stop you using more channels/control mechanisms, and these can be very useful for engaging additional actuators and autopilot modes.
 
 
-## Choosing Transmitter/Receivers
+## Choosing RC System Components
+
+You will need to select a transmitter/receiver pair that are compatible with each other. In addition, receivers have to be be [compatible with PX4](#compatible_receivers) and the flight controller hardware.
+
+Compatible radio systems are often sold together. For example, [FrSky Taranis X9D and FrSky X8R](https://hobbyking.com/en_us/frsky-2-4ghz-accst-taranis-x9d-plus-and-x8r-combo-digital-telemetry-radio-system-mode-2.html?___store=en_us) are a popular combination.
 
-You will need to select a transmitter and receiver that are compatible with each other. Compatible radio systems are often sold together. For example, this [Taranis X9D and FrSky X8R](https://hobbyking.com/en_us/frsky-2-4ghz-accst-taranis-x9d-plus-and-x8r-combo-digital-telemetry-radio-system-mode-2.html?___store=en_us) is a popular combination.
 
-> **Note** Before you can use the radio system you will need to *bind* them so that they communicate. Follow the binding instructions that come with your specific transmitter/receiver!
+### Transmitter/Receiver Pairs
 
-In addition, receivers have to be be compatible with PX4 and the flight controller hardware. The sections below list some of the receivers that have been validated for use with Flight Controller boards and PX4
+One of the most popular RC units is the *FrSky Taranis X9D*. It has an internal transmitter module can be used with the recommended *FrSky X4R-SB* (S-BUS, low delay) or *X4R* (PPM-Sum, legacy) receivers out of the box. It also has a custom radio transmitter module slot and customizable open source OpenTX Firmware. 

```

Comment:
The "X4R-SB* (S-BUS, low delay) or *X4R* (PPM-Sum, legacy) receivers out of the box" would seem to be older receivers. Are there better ones that are now recommended?

Reply:
I would recommend X4R-SB for use with PX4 (that's why I wrote it). Why do you consider them "old receivers"?
I sure know some transmitter/receiver pair which is potentially a lot better (namely TBS Crossfire micro) but also more expensive and I didn't test it with PX4 before, likely they are not supported yet.

Reply:
@MaEtUgR Thanks. I didn't check - I guessed they were older based on the low "number" compared to other receivers you can use. Dumb. Thanks for the confirmation. 
If there are other receivers we should include here, I'm sure you would have let me know.